src/DTS/Repository/DocDataRepository.php

```diff
@@ -39,7 +38,7 @@ public function newDocData($type, $key, $data) {
     );
   }
 
-  public function allVersions(string $docType, string $dataKey) {
+  public function allVersions(string $docType, string $dataKey): ?array {
```

Comment:
why the `?` ... this should always return an array ... though it might be empty ... i honestly haven't looked up php's stuff on what ?array would even mean ... i assume 'maybe an array' lol which isn't very helpful

Reply:
I was adding return types as I edited files without them. When I ran the tests after adding `: array` I got a failing test.

I figured at some point persistence was returning null, somewhere. I just took another look, it's a persistence spy in one test, doesn't return anything so I get an error when just "array" is the return type. I can fix it by making the spy return an empty array. I'll do that next time I'm in that file.